utils_linux.go

```diff
@@ -454,3 +462,14 @@ func startContainer(context *cli.Context, spec *specs.Spec, action CtAct, criuOp
 	}
 	return r.run(spec.Process)
 }
+
+/* setLinuxPersonality sets the Linux execution personality. For more information see the personality syscall documentation.
+   checkout getLinuxPersonalityFromStr() from libcontainer/specconv/spec_linux.go for type conversion.
+*/
```

Comment:
Please use `//` comments.

Reply:
@kolyshkin This is resolved in latest commit.